content/release-notes/2022-11-04.md

```diff
@@ -0,0 +1,17 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Frankfurt, Ohio, Singapore regions added.
+- UI: Branching support for the users with feature flag enabled.
+- UI: Support for postgres 15 keywords in query playground.
+- UI: Topbar is shown in side drawer on smaller screens.
+
+### Bug fixes
+
+- UI: Successful execution of multiqueries in query playground.
+- UI: The .pgpass instructions on reset password popup have the port number.
+- Control plane: users passing same oauth login challenge that have already been handled seconds time should now get redirect to final destination URL instead of 410 Gone error. 
```

Comment:
```suggestion
- Control Plane: Improved OAuth consent challenge handling. Submitting an OAuth consent challenge that was  handled previously now redirects a user to the final destination URL to complete the login or consent request instead of returning a `410 Gone` error.
```